Makefile

```
@@ -1,60 +1,67 @@
-VENV = venv
-PYTHON = $(VENV)/bin/python3
-PIP = $(VENV)/bin/pip
+PYTHON = python3
 
+VENV      = venv
+ACTIVATE := source $(VENV)/bin/activate
```

```suggestion
ACTIVATE := . $(VENV)/bin/activate
```

Based on comment made here: https://github.com/cowprotocol/ebbo/pull/53#discussion_r1252914896